stubs/requests/requests/sessions.pyi

```diff
@@ -57,6 +57,7 @@ _Params = Union[
     Iterable[Tuple[_ParamsMappingKeyType, _ParamsMappingValueType]],
     Union[Text, bytes],
 ]
+_SessionT = TypeVar("_SessionT", bound=Session)
```

Comment:
Nit: `bound` is not really necessary for annotated `self` types.

Reply:
I suppose you're right, though the mypy documentation (https://mypy.readthedocs.io/en/stable/more_types.html#precise-typing-of-alternative-constructors) says to use a bound. Removing bound here would be a nice simplification, since I'm pretty sure self types are by far the most common use case for bound typevars.

Reply:
We have a few similar unbound cases in typeshed already. mypy also seems to handle it correctly:

```python
from typing import TypeVar, Type

_T = TypeVar("_T")

class Foo:
    def bar(self: _T, x: _T): ...
    def baz(self: _T, x: Type[_T]): ...
    
Foo().bar("")  # expected Foo
Foo().baz("")  # expected Type[Foo]
```